app/src/main/res/xml/settings.xml

```diff
@@ -18,12 +18,15 @@
             app:title="@string/enable_background_playback_pip_mode"
             app:singleLineTitle="false"
             app:iconSpaceReserved="false" />
-        <SwitchPreferenceCompat
-            app:key="com.odysee.app.preference.userinterface.MediaAutoplay"
-            app:defaultValue="true"
+        <DropDownPreference
+            app:key="com.odysee.app.preference.userinterface.AutoplayMedia"
+            app:entries="@array/media_autoplay"
+            app:entryValues="@array/media_autoplay_values"
+            app:defaultValue="nothing_playing"
             app:title="@string/enable_autoplay"
             app:singleLineTitle="false"
-            app:iconSpaceReserved="false" />
+            app:iconSpaceReserved="false"
+            app:summary="%s" />
```

Comment:
Could this be moved after the title attribute? I see it better grouped. Please, do the same on the v29 folder

Reply:
Done